### DISCUSSION TITLE
Scrollto / Locate / Hightlight search results in column heading

### DISCUSSION THREAD

**Discussion by helt** [KevinVandy/material-react-table#960] [Q&A]
I have tables that have hundreds of columns. A frequent user task therefore is to locate a specfic column. I hoped that the search box would also highlight hits in the column header, but apparently, I either did not find the corresponding feature flag or it is not implemented. 

Somebody has an idea how to solve that issue?

**Comment by KevinVandy** [comment]
You can implement it yourself and use the column virtualizer scrollToIndex method to programmatically scroll to a column index

**Reply by imnasnainaec** [reply]
The documentation for this (https://www.material-react-table.com/docs/guides/virtualization#access-underlying-virtualizer-instances) has `useRef<Virtualizer>`, but as I try to implement, it seems that `useRef<MRT_RowVirtualizer>` (or correspondingly `useRef<MRT_ColumnVirtualizer>`) is expected.

**Reply by imnasnainaec** [reply]
Submitted pr to fix the docs: https://github.com/KevinVandy/material-react-table/pull/1010
